cmd/run/run.go

```diff
@@ -412,6 +415,29 @@ func run(_ *cobra.Command, _ []string) {
 	}
 }
 
+func newCertWatcher(ctx context.Context, logger *logger.ZapLogger, certPath, keyPath string) (*certwatcher.CertWatcher, error) {
+	if _, err := os.Stat(certPath); err != nil {
+		return nil, err
+	}
+	if _, err := os.Stat(keyPath); err != nil {
+		return nil, err
+	}
```

Comment:
Do we need this code? The presence of these files is being checked by https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/certwatcher/certwatcher.go#L139

Reply:
Yeah, I think you're right. Fixed this